susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1311,6 +1311,10 @@
     'almalinux-8-x86_64-uyuni' : {
         'BASECHANNEL' : 'almalinux8-x86_64', 'PKGLIST' : RES8 + RES8_X86,
         'DEST' : '/srv/www/htdocs/pub/repositories/almalinux/8/bootstrap/'
+    },    
+    'rockylinux-8-x86_64-uyuni' : {
+        'BASECHANNEL' : 'rockylinux-x86_64', 'PKGLIST' : RES8 + RES8_X86,
```

Comment:
The basechannel should actually be rockylinux8-x86_64 or it won't be able to create the bootstrap repo.

Reply:
Seems to be fixed.